### PR TITLE
[stable10] Backport of Replace notification with confirm dialog for u…

### PR DESCRIPTION
**Pull request by sharidas**
## Description

Replace the notification when a user/group is deleted from the users page, with a confirmation dialog.
- The confirmation dialog will be popped up, when a delete action is triggered from the users page on a user or a group.
- If user clicks button yes of the confirmation dialog, the user/group would be deleted. 
- If button no is clicked of the confirmation dialog, the delete operation will not happen, for the user/group.

## Related Issue

- Fixes <issue_link>

## Motivation and Context

Replace the notification when a user/group is deleted from the users page, with a confirmation dialog.

## How Has This Been Tested?

- [x] All the webUI tests were conducted from the users page.
- [x] Did simple tests in the webUI by creating users and deleting them. Verified with the occ command if the users were deleted.
- [x] Did simple tests in the webUI by creating users and cancling the delete operation. Verified with the occ command if the users were still available.
- [x] Created groups and deleted them from webUI. Verified the same with occ command.
- [x] Created groups and cancelled the delete operation done on them. Verified the same with occ command, if the groups were still available.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [x] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

**Comment by sharidas** [comment]
Original PR: https://github.com/owncloud/user_management/pull/94

File: settings/js/users/deleteHandler.js [hunk 1]
```diff
@@ -44,27 +44,6 @@ DeleteHandler.TIMEOUT_MS = 7000;
  */
 DeleteHandler.prototype._timeout = null;
 
-/**
- * The function to be called after successfully marking the object for deletion
```
Comment (phil-davis) [inline settings/js/users/deleteHandler.js]: You didn't delete these comment sections in user_management.
I guess these deletes should be "cross-ported to user_management...

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by phil-davis — APPROVED** [review]
LGTM

File: settings/js/users/deleteHandler.js (hunk 1 above)
Comment (sharidas) [reply settings/js/users/deleteHandler.js]: Yes I will create a separate pr for the user_management.

**Comment by phil-davis** [comment]
cross-port of the extra deleted comment code is done in https://github.com/owncloud/user_management/pull/111

**Pull request merged**